perfkitbenchmarker/benchmark_sets.py

```diff
@@ -190,7 +191,7 @@ def GetBenchmarksFromFlags():
     else:
       benchmark_names.add(benchmark)
 
-  # Expand recursive sets
+# Expand recursive sets
```

Comment:
Are you manually running flake8? If you've installed the hooks in this repository, they should be running it through tox, which is configured to use a specific version of flake8: https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/c4289c6035f95f6e3964273ebf43dd1788a25a9a/tox.ini#L27
We have an open issue to make changes to support newer versions of pep8 (#797), but for now `tox -r -e flake8` should re-download required packages at the specified versions and execute flake8 with them.